charactersheet.html

```diff
@@ -0,0 +1,579 @@
+<!DOCTYPE html>
```

Comment:
Avoid inline styles unless they are absolutely necessary

Most of your inline styles could have gone on classes